content/blog/set-up-apache-skywalking-for-k8s-and-vm-monitoring-in-hpe-greenlake-private-cloud.md

```diff
@@ -0,0 +1,382 @@
+                          meta.helm.sh/release-namespace: skywalking
+Selector:                 app=skywalking,component=ui,release=skywalking
+Type:                     NodePort
+IP:                       10.102.186.131
+Port:                     <unset>  80/TCP
+TargetPort:               8080/TCP
+NodePort:                 <unset>  32748/TCP
+Endpoints:                10.192.7.25:8080
+Session Affinity:         None
+External Traffic Policy:  Cluster
+Events:                   <none>
+```
+
+As shown in the ***Annotations*** section of the service description above, the SkyWalking UI can then be accessed in the browser by typing the address *gl2-caas.gl-hpe.local:10037*: 
+
+![](/img/sw-ui.png)
+
+### Deploy a sample application: *SpringBoot*
+
+A﻿s the first demo application, create a *SpingBoot* Web app that provides a REST endpoint **/message** to print some nice message. Then generate a *jar* package using the [Apache Maven](https://maven.apache.org/what-is-maven.html) command *mvn* with the *pom.xml* file:
+
+```markdown
+├── Dockerfile
+├── Dockerfile.agentless
+├── pom.xml
+├── README.md
+├── src
+│   ├── main
+│   └── test
+└── target
+    ├── springboot-k8s-demo.jar
+    └── test-classes
+
+$﻿ cat pom.xml
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+	<parent>
+		<groupId>org.springframework.boot</groupId>
+		<artifactId>spring-boot-starter-parent</artifactId>
+		<version>2.6.1</version>
+		<relativePath/> <!-- lookup parent from repository -->
+	</parent>
+	<groupId>com.javatechie</groupId>
+	<artifactId>springboot-k8s-demo</artifactId>
+	<version>0.0.1-SNAPSHOT</version>
+	<name>springboot-k8s-demo</name>
+	<description>Demo project for Spring Boot</description>
+	<properties>
+		<java.version>1.8</java.version>
+	</properties>
+	<dependencies>
+		<dependency>
+			<groupId>org.springframework.boot</groupId>
+			<artifactId>spring-boot-starter-web</artifactId>
+		</dependency>
+
+		<dependency>
+			<groupId>org.springframework.boot</groupId>
+			<artifactId>spring-boot-starter-test</artifactId>
+			<scope>test</scope>
+		</dependency>
+	</dependencies>
+
+	<build>
+		<plugins>
+			<plugin>
+				<groupId>org.springframework.boot</groupId>
+				<artifactId>spring-boot-maven-plugin</artifactId>
+			</plugin>
+		</plugins>
+		<finalName>springboot-k8s-demo</finalName>
+	</build>
+
+</project>
+
+$﻿ mvn compile
+$﻿ mvn package
+$ ls target/springboot-k8s-demo.jar
+target/springboot-k8s-demo.jar
+```
+
+You can see the *jar* package *springboot-k8s-demo.jar* is built to the *target* folder. B﻿y building a *Docker* image using this generated *jar* file, the *SpringBoot* Web app can be easily deployed as a containerized application to the Kubernetes cluster.
```

Comment:
I think it would read more clearly if you change "built to the target folder" to "created in the target folder." If you don't care for that, at least change "to" to "in". Similarly, "to the Kubernetes cluster." should be "in the Kubernetes cluster."